src/isl/util.rs

```diff
@@ -35,12 +35,16 @@ impl Range {
         use RangeBoundaryValueType::*;
         match self {
             Range::Integer(start, end) => {
-                let unwrapped_value = value.as_any_int().unwrap();
+                let value = value.as_any_int().ok_or_else(|| {
+                    return invalid_schema_error_raw(
+                        "Integer ranges can only have integer value for validation",
+                    );
+                })?;
```

Comment:
Here and elsewhere in the PR, you can simplify this to:
```rust
let value = value.as_any_int().ok_or_else(|| invalid_schema_error_raw("..."))?;
```
(That is: you can remove the explicit `return` and braces.)